IFC4x3/PropertyConstants/e/EXTERNALCOMPANY_2KTrw1IIzBqxUGLKizr1oq.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="EXTERNALCOMPANY_2KTrw1IIzBqxUGLKizr1oq" Name="EXTERNALCOMPANY" UniqueId="94775e81-492f-4bd3-b790-554b3dd41cb4" />
+
```

Comment:
Documentation missing. See also other comment regarding ConstructionCompany.

Reply:
I have erased Enum. adb5fdd751ea322bf0649365b080a81b2a458a62

Reply:
This enum is still here.

Reply:
Done Here. a14c639b6c4248c34ffadf5422b6425f466a4277